### PR TITLE
(Re-)adding GT CSE Faculty

### PR DESCRIPTION
**Pull request by catalyurek**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -18777,6 +18779,7 @@ Tobias Scheffer,University of Potsdam,https://www.uni-potsdam.de/en/cs-ml/staff/
 Tobias Schreck,Graz University of Technology,https://www.tugraz.at/institute/cgv/schreck,9cJsSIIAAAAJ
 Tobias Weinzierl,Durham University,https://www.dur.ac.uk/computer.science/staff/profile/?id=12329,pRrvub4AAAAJ
 Tobias Wrigstad,Uppsala University,http://wrigstad.com,EfO5KRQAAAAJ
+Tobin Isaac,Georgia Institute of Technology,https://triquadtethex.org,SpDPpqgAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Problem with https on this site.

File: csrankings.csv [hunk 2]
```diff
@@ -5718,6 +5719,7 @@ Felix A. Wichmann,University of Tübingen,https://uni-tuebingen.de/en/faculties/
 Felix Brandt 0001,TU Munich,http://dss.in.tum.de/staff/brandt,fVnXdaQAAAAJ
 Felix C. Freiling,University of Erlangen–Nuremberg,https://www.cs1.tf.fau.de/person/felix-freiling,NOSCHOLARPAGE
 Felix Freitag,Polytechnic University of Catalonia,http://people.ac.upc.edu/felix,6wJ4ZdQAAAAJ
+Felix J. Herrmann,Georgia Institute of Technology,http://www.slim.eos.ubc.ca,hw23CTQAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: In ECE; no obvious cross-appointment with CS.

**Comment by emeryberger** [comment]
See above comments.

File: csrankings.csv (hunk 1 above)
Comment (catalyurek) [reply csrankings.csv]: Sorry I will ask him to fix his personal web site. But here is his official GT CSE Faculty page: https://www.cse.gatech.edu/people/tobin-isaac

File: csrankings.csv (hunk 2 above)
Comment (catalyurek) [reply csrankings.csv]: GT School of CSE faculty is listed here: https://www.cse.gatech.edu/people/faculty  and Felix's page is here: https://www.cse.gatech.edu/people/felix-herrmann  He  apparently didn't update his personal web page recently. He has joint appointment with our school (his office and lab is in the School of CSE). He can advise CS and CSE PhD students with this appointment. I wanted to be inclusive.

File: csrankings.csv (hunk 1 above)
Comment (catalyurek) [reply csrankings.csv]: Sorry it seems like http://triquadtethex.org (non-secure) link works. let me know if you want me resubmit merge request, or if you merge and fix together.

Comment (emeryberger) [reply csrankings.csv]: If you can update, that'd be easiest.

Comment (catalyurek) [reply csrankings.csv]: fixed

**Pull request merged**